File: haskell/fizzbuzz.hs
fizzbuzz :: [String]
fizzbuzz = [if n `mod` 3 == 0 && n `mod` 5 == 0 then "FizzBuzz" else
            if n `mod` 3  == 0 then "Fizz" else
            if n `mod` 5  == 0 then "Buzz" else
            show n | n <- [1..100]]


main :: IO ()
main =  mapM_ putStrLn fizzbuzz

File: julia/fizzbuzz.jl
for x=1:100 
    if mod(x,3) == 0 && mod(x,5) == 0
        println("FizzBuzz")
    elseif mod(x,3) == 0
        println("Fizz")
    elseif mod(x,5) == 0
        println("Buzz")
    else
        println(x)
    end
end